ttnn/ttnn/operations/normalization.py

```diff
@@ -197,12 +199,81 @@ def rms_norm(input_tensor: ttnn.Tensor, weight: ttnn.Tensor, *, epsilon: float =
     return output_tensor
 
 
+# group norm helper function
+def determine_expected_group_norm_sharded_config_and_grid_size(
```

Comment:
@arakhmati to review this when he's back on Monday

Reply:
Group norm should be a class in TTNN and this function should go there along with other helper functions for group norm. Is it okay to check this in for now? Once the GN class is added, this will be moved.

Reply:
Why does group norm need to be a class?

Reply:
GN takes in "config" tensors (masks for zeroing out portions of tiles) that are initialized at module building time and passed to GN op during runtime. This is similar to conv and maxpool that have a class to handle config tensors.